dashboard/app/views/musiclab/index.html.haml

```diff
@@ -3,4 +3,5 @@
 
 - content_for(:head) do
   %script{src: webpack_asset_path('js/googleblockly.js')}
+  %script{src: webpack_asset_path("js/#{js_locale}/music_locale.js")}
```

Comment:
Just curious, does this change work for all four URLs?

http://localhost-studio.code.org:3000/levels/43609 (ID may vary)
http://localhost-studio.code.org:3000/projectbeats
http://localhost-studio.code.org:3000/projectbeats?load-progression=true&local-progression=true
http://localhost-studio.code.org:3000/s/allthethings/lessons/46/levels/2

Reply:
Yep! This change specifically enables localization for the standalone URL; localization within the allthethings script is already taken care of by _apps_dependencies.html.haml.